src/test/kerberos/meson.build

```diff
@@ -7,6 +7,7 @@ tap_tests += {
   ],
   'env' : {
     'with_gssapi': gssapi.found() ? 'yes' : 'no',
+    'PG_TEST_EXTRA': get_option('PG_TEST_EXTRA'),
```

Comment:
I think it'd be better to just add that to env once globally, in the toplevel meson.build.